realsense2_camera/include/t265_realsense_node.h

```diff
@@ -19,13 +20,26 @@ namespace realsense2_camera
 
         private:
             void initializeOdometryInput();
+            void setupMapReutilization();
             void setupSubscribers();
             void handleWarning();   
             void odom_in_callback(const nav_msgs::Odometry::ConstPtr& msg);
             void warningDiagnostic (diagnostic_updater::DiagnosticStatusWrapper &stat);
+            bool importLocalizationMap(const std::string &localization_file);
+            bool exportLocalizationMap(const std::string &localization_file);
             diagnostic_updater::Updater callback_updater;
 
+            // Service callbacks
+            bool saveRelocalizationMapSrv(realsense2_camera::MapPathString::Request &req,
+                                          realsense2_camera::MapPathString::Response &res);
+
+            // Helper functions to import/export binary map files
+            std::vector<uint8_t> bytesFromRawFile(const std::string &filename);
+            void rawFileFromBytes(const std::string &filename, const std::vector<uint8_t> &bytes);

```

Comment:
Convert the T265 map from bytes into binary, so we can save locally. This was inpired from the function `bytes_from_raw_file` in https://github.com/IntelRealSense/librealsense/blob/master/examples/ar-advanced/rs-ar-advanced.cpp